src/zen/websocket_server.cpp

```diff
@@ -556,18 +556,19 @@ class WsHandler
                 CScCertificate topQualCert;
                 uint256 blockHash;
 
+                chainTopQualityCert.push_back(Pair("quality", topQualityCertQuality));
+                chainTopQualityCert.push_back(Pair("certHash", sidechainInfo.lastTopQualityCertHash.GetHex()));
+                chainTopQualityCert.push_back(Pair("epoch", sidechainInfo.lastTopQualityCertReferencedEpoch));
+
                 if (GetCertificate(sidechainInfo.lastTopQualityCertHash, topQualCert, blockHash, true)) {
```

Comment:
Passing allowSlow = true implies loading and parsing the full block from disk. Considering that now the sidechain cannot expect to always have the rawCertificateHex returned (eg. it will not be returned if the cert had no bt outputs and change was already spent) why not always avoiding this expensive operation?

Reply:
After discussion it has been decided to remove the call to GetCertificate()